### PR TITLE
Add the gulps

### PR DESCRIPTION
**Pull request by ricardoom**
I've added the gulps for convenience. to test, run `npm install` the from the root run `gulp`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,20 @@
   },
   "main": "index.js",
   "dependencies": {
-    "node-sass": "3.8"
+    "node-sass": "4.6.0"
   },
   "devDependencies": {
+    "browser-sync": "^2.24.3",
```
Comment (EdwinGuzman) [inline package.json]: Might be good to remove the carets to have specific versions.

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    .pipe(sass(sassOptions).on('error', sass.logError))
+    .pipe(sourcemaps.write('./'))
+    .pipe(gulp.dest(output));
+});
+
+gulp.task('watch', function () {
+  gulp.watch('sass/**/*.scss', ['sass']);
+});
+
+gulp.task('default', ['sass', 'watch']);
```
Comment (EdwinGuzman) [inline gulpfile.js]: I'm not sure if you want the `watch` task to be part of the default. It'll be watching and the user may not know that that is happening.

File: package.json (hunk 1 above)
Comment (ricardoom) [reply package.json]: gotcha, I'll also remove browser sync, since its not ready for prime time...

File: gulpfile.js (hunk 2 above)
Comment (ktp242) [reply gulpfile.js]: Having the documentation updated will be great!!

**Comment by ricardoom** [comment]
thanks. merging.

**Pull request merged**